conf/default-config-options.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @fileoverview Default config options
+ * @author Teddy Katz
+ */
+
+"use strict";
+
+/**
+ * Freezes an object and all its nested properties
+ * @param {Object} obj The object to deeply freeze
+ * @returns {Object} `obj` after freezing it
+ */
+function deepFreeze(obj) {
+    if (obj === null || typeof obj !== "object") {
+        return obj;
+    }
+
+    Object.keys(obj).map(key => obj[key]).forEach(deepFreeze);
+    return Object.freeze(obj);
+}
+
+module.exports = deepFreeze({
```

Comment:
If I'm not mistaken, due to node's `require` cache, the object is just created once and cached afterwards, so I think it would just be once on load.

I couldn't find the performance issue in the issue tracker, but I did find [this](https://bugs.chromium.org/p/chromium/issues/detail?id=115960), which has been fixed in modern Node versions. I'm not sure if that's the same issue we were encountering though.